examples/lessons/13/app.js

```diff
@@ -166,38 +169,46 @@ const animationLoop = new AnimationLoop({
       }),
       program: fragmentLightingProgram,
```

Comment:
Should we stop making programs in advance and just pass in the shaders to the Model constructors to avoid the sharing issue (which seems to be an overly ambitious optimization unrelated to the main exposition of this example, i.e. fragment lighting)?

Reply:
Sounds good.